features/HTTP.feature

```diff
@@ -1575,3 +1575,7 @@ Feature: Test HTTP API
     And There is no specifications for index "kuzzle-test-index" and collection "kuzzle-collection-test"
     When I delete the specifications again for index "kuzzle-test-index" and collection "kuzzle-collection-test"
     Then There is no error message
+
+  @usingHttp
+  Scenario: Get authentication strategies
+    Then I get the registrated authentication strategies
```

Comment:
Additionally to the retrieval of authentication strategies, I wonder if we shouldn't verify if they are well-formed/if we retrieved what is expected by default.